Makefile

```diff
@@ -257,6 +257,24 @@ clean-for-device-build:
 .PHONY: clean-test-with-device-farm
 clean-test-with-device-farm: clean-for-device-build test-with-device-farm
 
+
+.PHONY: test-with-firebase
+test-with-firebase:
+# Ignore the build part as we have build in fastlane, recomment out these part if test locally.
+# 	xcodebuild \
+# 		-workspace Apps/Apps.xcworkspace \
+# 		-scheme $(SCHEME) \
+# 		-enableCodeCoverage YES \
+# 		-configuration $(CONFIGURATION) \
+# 		-sdk iphoneos build-for-testing;
+	cd ${BUILD_DIR}/Build/Products; \
+	zip -r ${APP_NAME}.zip $(CONFIGURATION)-iphoneos $(SCHEME)_iphoneos*.xctestrun;
```

Comment:
We can replace this two commands with additional argument to the `scan` fastlane action – `should_zip_build_products`